### PR TITLE
Updated the version of the test framework

### PR DESCRIPTION
**Pull request by anchouls**
The testing framework is only loaded into subprojects with the Practice suffix.

### CODE REVIEW

File: InliningCode/MiddleManCodeSmell/PracticePractice/test/Tests.kt [hunk 1]
```diff
@@ -41,8 +38,4 @@ class MiddleManTest : BaseIjTestClass() {
             "Please, invoke \"dataProvider.fetchData()\" inside \"Client::processData\" method."
         }
     }
-
-    private fun hasClass(className: String) = ApplicationManager.getApplication().runReadAction<Boolean> {
```
Comment (onewhl) [inline InliningCode/MiddleManCodeSmell/PracticePractice/test/Tests.kt:45]: Don't we need this check anymore?

File: InliningCode/MiddleManCodeSmell/lesson-info.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ type: framework
 custom_name: Middle Man code smell
 content:
   - Theory
-  - Practice
+  - PracticePractice
```
Comment (onewhl) [inline InliningCode/MiddleManCodeSmell/lesson-info.yaml:5]: Why do we name it as PracticePractice?

File: RefactoringAndItsPurpose/Practice/RefactoringAndItsPurposePractice/test/IdentifyIssuesInCodeSnippetTest.kt [hunk 3]
```diff
@@ -55,8 +52,4 @@ class IdentifyIssuesInCodeSnippetTest : BaseIjTestClass() {
             "Please, rename \"w\" parameter to \"width\""
         }
     }
-
-    private fun hasParameter(parameterName: String) = ApplicationManager.getApplication().runReadAction<Boolean> {
```
Comment (onewhl) [inline RefactoringAndItsPurpose/Practice/RefactoringAndItsPurposePractice/test/IdentifyIssuesInCodeSnippetTest.kt:59]: Don't we need this check anymore?

**Review by onewhl — APPROVED** [review]
Thanks! Please, take a look at the comments.

File: RefactoringAndItsPurpose/Practice/RefactoringAndItsPurposePractice/test/IdentifyIssuesInCodeSnippetTest.kt (hunk 3 above)
Comment (anchouls) [reply RefactoringAndItsPurpose/Practice/RefactoringAndItsPurposePractice/test/IdentifyIssuesInCodeSnippetTest.kt:59]: This has been put into the test framework

**Pull request merged**